README.md

```diff
@@ -186,23 +186,6 @@ root@9feae5e5e001:/src/conjur/cucumber/api# cucumber
 root@9feae5e5e001:/src/conjur# cucumber -r cucumber/api/features/support -r cucumber/api/features/step_definitions cucumber/api/features/resource_list.feature
 ```
 
-## Documentation site
```

Comment:
What do you think about linking to the new `conjur-org` repo to let people know that docs have moved?

Reply:
The task breakdown has me adding a README into the /docs folder (which is what I am currently working on) or would you rather it be here?

Reply:
Nope.  Let's add a readme to `docs/` as stated in the story.